source/features/extend-status-labels.js

```diff
@@ -6,17 +6,27 @@ export default function () {
 	const lastActionRef = select.all(`
 		.discussion-item-closed [href*="/pull/"],
 		.discussion-item-closed code,
-		.discussion-item-reopened
+		.discussion-item-reopened,
+		.discussion-item .discussion-item-entity
```

Comment:
This should probably be `.discussion-item-merged [href*="/commit/"]` to be more specific and to skip the `.closest()` call